config/routes.rb

```diff
@@ -26,6 +24,14 @@
     end
 
     resources :events, :only => [:index]
+
+    scope module: :agents do
```

Comment:
Almost, as far as I know the only difference are the route names. 
For `namespace`: `agent_agents_dry_runs` and `agents_dry_runs`.
For `scope`: `agent_dry_runs` and `dry_runs`.